consumer/nf_management.go

```diff
@@ -39,7 +39,8 @@ func BuildNFInstance(ausfContext *ausf_context.AUSFContext) (profile models.NfPr
 }
 
 // func SendRegisterNFInstance(nrfUri, nfInstanceId string, profile models.NfProfile) (resouceNrfUri string,
-//    retrieveNfInstanceID string, err error) {
+//
+//	retrieveNfInstanceID string, err error) {
```

Comment:
@gruyaume, actually, should we remove all these lines?

Reply:
Well probably but the goal here was simply to satisfy `gofmt`. All over the code there are comments that are likely not wanted but I didn't want to go through this activity here.